code/game/turfs/turf.dm

```diff
@@ -80,6 +80,13 @@
 	if (opacity)
 		has_opaque_atom = TRUE
 
+	var/temp_list = list()
```

Comment:
This should be behind some early check, crunching all this for all turfs is pretty wasteful.